src/app/features/genes/components/gene-soe-charts/gene-soe-charts.component.scss

```diff
@@ -4,15 +4,12 @@
 @import '../../../../../styles/mixins';
 
 .gene-score-charts {
-  display: flex;
-  flex-direction: row;
-  flex-wrap: wrap;
-  justify-content: center;
-  gap: 20px;
+  display: grid;
```

Comment:
Use a grid to ensure that all plots have the same width, even when wrapped onto multiple rows